### ISSUE TITLE
Would this work with automated shutter 

### ISSUE THREAD

**Issue by atv2016** [PlusPlus-ua/ha_tuya_ble#15]
￡11.96  45％ Off | Tuya USB Bluetooth Smart Adjustable Blinds Motor Charging Plantation Shutters Window Smart Life Alexa Google Home Voice Control
https://a.aliexpress.com/_mMbKgla

Basically a switch, I’m kinda hoping it would work?

**Comment by PlusPlus-ua** [comment]
It should work it the device uses BLE, not Bluetooth Mesh protocol.

Unfortunately, It's impossible to understand from Ali descriptions. I've bought a small Tuya Bluetooth lock, and it turned out to be Bluetooth Mesh.

**Comment by atv2016** [comment]
I didn’t know about Bluetooth mesh. Is that regular Bluetooth?

**Comment by PlusPlus-ua** [comment]
> I didn’t know about Bluetooth mesh. Is that regular Bluetooth?

No, It's based on BLE as far as I know, but uses only part of BLE functionality to provide a mesh network, like ZigBee does. Anyway, Home Assistant currently has no support for BT mesh in official Bluetooth integration. Any of Tuya gateways which has Bluetooth interface support both BLE and BT mesh protocols.

**Comment by atv2016** [comment]
Cool good to know. I got the little tuya hun with it just in case I can’t get it to work, failing that it seems to have a RF interface.
________________________________

**Comment by atv2016** [comment]
Well it seems it has added it automatically overnight, however there are no exposed entities. It seems the integration got my cloud Id and access credentials automatically so I just clicked configure, submit and voila. But yeah no entities that I can control.

Or maybe I should double check that, did it get the creds from the tuya integration maybe? Or local tuya in the past?
________________________________

**Comment by atv2016** [comment]
Do I need to set it up with the little tuya hub first maybe?

To note it also became available in the usual tuya app, and I can control it from there. Subsequently it also became available in the HA tuya integration, that displays a control entity but is greyed out sigh.

And last but not least even though I can control the shutters from the tuya app, I cannot call the scenes from HA.

I ordered a broad link pro just in case from AliExpress, this thing has Rf as well.
________________________________

**Comment by mcastrigno** [comment]
@atv2016 I have a similar device. I am very interested in your progress attempting to use this integration. I am VERY new to HA but not software development. What can you tell me about your progress here? My device can be controlled via the SmartLife App via BlueTooth. With my phone offline I can still control it so I feel like "there is hope" for local control.
https://www.a-okmotor.com/am20-indoor-smart-automated-plantation-shutters-motor_p59.html
